src/utils/discord_util.py

```diff
@@ -1,13 +1,47 @@
 """
     Discord Utility module
 """
-
+from discord import Intents, Client
 from .command_interface import CommandInterface
 
+intents = Intents.default()
+intents.message_content = True
+client = Client(intents=intents)
+
+
 class DiscordUtil(CommandInterface):
     """
-        A class for performing all logic to interact with Discord as a bot
+    A class for performing all logic to interact with Discord as a bot
     """
 
     def __init__(self, discord_auth_token):
+        self.token = discord_auth_token
+
+        self.run()
```

Comment:
In this case, we want to run() only when called externally and not in the constructor. `app.py` is already calling run() so this would lead to it running twice. It could be worth adding a `is_running` variable to the object to prevent `run()` from being executed more than once.